Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPILocalizationCocoa.mm

```diff
@@ -65,7 +67,11 @@
         });
     }
 
-    return [localization localizedStringForKey:messageName withPlaceholders:substitutionsArray];
+    Vector<String> substitutionsVector;
+    for (id string in substitutionsArray)
+        substitutionsVector.append(string);
```

Comment:
Need `#import <wtf/cocoa/VectorCocoa.h>` but this will help:
```suggestion
    auto substitutionsVector = makeVector<String>(substitutionsArray);
```